autoprecompiles/src/lib.rs

```diff
@@ -293,6 +294,9 @@ pub enum VMBusInteraction<T> {
 pub const EXECUTION_BUS_ID: u64 = 0;
 pub const MEMORY_BUS_ID: u64 = 1;
 pub const PC_LOOKUP_BUS_ID: u64 = 2;
+pub const VARIABLE_RANGE_CHECKER_BUS_ID: u64 = 3;
+pub const BITWISE_LOOKUP_BUS_ID: u64 = 6;
+pub const TUPLE_RANGE_CHECKER_BUS_ID: u64 = 7;
```

Comment:
Btw this won't work for reth because its more complex config changes the bus id of TUPLE_RANGE_CHECKER_BUS, but we can make that dynamic in a different PR